app/src/main/java/com/globant/example/mentorapp/subscriberDetails/domain/model/RepositoryModel.java

```diff
@@ -0,0 +1,24 @@
+package com.globant.example.mentorapp.subscriberDetails.domain.model;
+
+/**
+ * Represents a model of repository basic information.
+ * Created by oscar.amezquita on 12/07/2017.
+ */
+public class RepositoryModel {
```

Comment:
Isn't it a view model for Repository? A domain model is implemented mainly by Gson.
Also, you're using it in your RepositoryAdapter